src/libs/time.js

```diff
@@ -0,0 +1,14 @@
+function getIntlLocales() {
+  const lang = window.getLang();
+  const locales = [lang.locale].concat(lang.fallback || []);
+  // Intl expects '-' in locales, such as "en-GB"
+  return locales.map(l => l.replace(/_/g, '-'));
+}
+
+export function getTimeFormatter(format = {}) {
+  return Intl.DateTimeFormat(getIntlLocales(), format);
+}
+
+export function stripTimeZone(utcString) {
+  return utcString.substring(0, 19);
```

Comment:
nitpick: maybe `isoString` would be a more correct name for this parameter, as the datetime could actually be expressed in any timezone.

Reply:
Indeed, and UTCString has a completely different meaning for `Date` objects so this was confusing.
I've changed it, thanks.